### DISCUSSION TITLE
Compatibility with Commeo Multi-Send

### DISCUSSION THREAD

**Discussion by Jbj1234** [Kannix2005/homeassistant-selve#15] [Q&A]
Hello, thanks for your work on this project!

I live in Australia where Selve are a niche brand and I am not able to find a retailer who is able to sell/ship the USB-RF dongle to me. The Selve Home Server range are also not for sale here. 

I have a Commeo Multi-Send, which I can attach to a windows PC using the micro-USB port. This allows me to program and control the shutters using the Selve Windows software. 

Do you have any advice or idea on how I would be able to use the Multi-Send device with this home assistant integration, instead of the USB RF?  

Unfortunately the Multi-Send is not recognised at the moment when running the integration set up flow. 

Thanks again for your work on this project and your insight!

**Comment by Kannix2005** [comment]
Can you tell me how the MultiSend is recognized under Windows? Is it a serial device? And which Selve software are you using? I don´t know this one yet.

**Reply by Jbj1234** [reply]
Thank you for the response, I really appreciate it.

The MultiSend is recognised as a serial device via COM port. The Selve software is Commeo MultiSend Konfigtool which appears to install a .inf driver 'SELVE_Commeo_MultiSend.inf' to 
C:\Windows\System32\DriverStore\FileRepository\selve_commeo_multisend.inf_amd64_e2904fa359245998 

 https://www.selve.de/en/Service-downloads/Software/ . 

Driver details on Device Manager:
Name	SELVE Commeo MultiSend (COM3)
Status	OK
PNP Device ID	USB\VID_2047&PID_09B2\72B69A4618002900
Baud Rate	9600
Bits/Byte	8
Stop Bits	1
Parity	None
Busy	Yes
Driver	C:\WINDOWS\SYSTEM32\DRIVERS\USBSER.SYS (10.0.19041.2130, 79.50 KB (81,408 bytes), 12/10/2022 6:31 PM)
Class GUID	{4D36E978-E325-11CE-BFC1-08002BE10318}
Device ID	USB\VID_2047&PID_09B2\72B69A4618002900

Please let me know if I can provide any further information. Thank you again for your help.

**Comment by Kannix2005** [comment]
That sounds like the MultiSend uses plain serial communication with supposedly AT-commands. I´d have to have a MultiSend to sniff the packages and implement a communication protocol. 
It´s a rather exotic use-case and rather expensive on my side :( 
Best would be if you can find a USB-RF gateway.

**Comment by frankfrommelt** [comment]
You can buy the Selve USB-RF gateways at eBay.
